src/assets/styles/variables.postcss

```diff
@@ -33,7 +34,7 @@
 
 :root {
   --tooltipBoxShadow: 0 5px 10px 3px #0003;
-  --headerBoxShadow: 0 0 5px 0 #999;
+  --headerBoxShadow: 0 1px 4px rgba(0, 0, 0, 0.13);
```

Comment:
If I recall correctly, the new design was supposed to substitute most shadows with borders.